rareapi/models/subscription.py

```diff
@@ -0,0 +1,9 @@
+"""Subscription Model Module"""
+from django.db import models
+
+class Subscription(models.Model):
+    """Subscription Model"""
+    follower_id = models.ForeignKey("RareUser", on_delete=models.CASCADE, related_name='follower_id')
```

Comment:
I would suggest changing the related name on both of these to 'subscriptions' or something like that...